card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  font-size: var(--description-font-size);
+  font-family: var(--font-family);
+  color: var(--outline-grey);
+  /* border: 1px solid var(--outline-grey); */
+  padding: 40px;
+  gap: 20px;
+  /* border-radius: 15px; */
+}
+
+.content-section {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.name-section {
+  margin-top: 20px;
+}
+
+.name-section-worker {
+  font-size: var(--base-font-size);
+  font-family: var(--font-family);
+  color: var(--black);
+  font-weight: var(--font-bold);
+  padding-right: 10px;
+}
+
+.star {
+  padding-right: 10px;
+}
+
+.star-span {
+  text-decoration: underline dotted 1px;
+  text-underline-offset: 5px;
+}
+
+.price {
```

Comment:
apply task 1 for all class names

Reply:
Can you explain what is needed to changed? @aravindv97

Reply:
Task 1 - Add component name prefix (card-content-section)

Reply:
**Resolved:** Added all prefix to the components and deleted unused components.